tests/data/openmathinstruct.test

```diff
@@ -1,100 +1,100 @@
-{"question": "What is the sum of the coordinates of the midpoint of the segment with endpoints $(8, 5)$ and $(2, -1)$?", "expected_answer": "7", "predicted_answer": "7", "error_message": "<not_executed>", "is_correct": true, "generation_type": "masked_reference_solution", "dataset": "math", "generation": "Let's first obtain the midpoint $(x, y)$ of the segment with endpoints $(8, 5)$ and $(2, -1)$. The midpoint formula is \\[(x, y) = \\left(\\frac{8 + 2}{2}, \\frac{5 - 1}{2}\\right) = (5, 2).\\]\n\nNow, we compute the sum of the coordinates of the midpoint: $x + y = 5 + 2 = 7$.\n\nTherefore, the sum of the coordinates of the midpoint of the segment with endpoints $(8, 5)$ and $(2, -1)$ is $\\boxed{7}$."}
-{"question": "Harold's fancy wrapping paper can wrap 5 shirt boxes or 3 XL boxes.  For the upcoming holiday, he has 20 shirt boxes to wrap and 12 XL boxes to wrap.  If each roll of wrapping paper costs $4.00 per roll, how much will he spend to wrap all of the boxes?", "expected_answer": "32", "predicted_answer": "32", "error_message": "", "is_correct": true, "generation_type": "masked_reference_solution", "dataset": "gsm8k", "generation": "Let's solve this problem using Python code.\n<llm-code>\nshirt_boxes = 20\nx_l_boxes = 12\npaper_roll_for_shirt_boxes = 5\npaper_roll_for_x_l_boxes = 3\nnumber_of_paper_rolls_for_shirt_boxes = shirt_boxes / paper_roll_for_shirt_boxes\nnumber_of_paper_rolls_for_x_l_boxes = x_l_boxes / paper_roll_for_x_l_boxes\ntotal_number_of_paper_rolls = number_of_paper_rolls_for_shirt_boxes + number_of_paper_rolls_for_x_l_boxes\ncost_per_paper_roll = 4.0\ntotal_cost = total_number_of_paper_rolls * cost_per_paper_roll\ntotal_cost\n</llm-code>\n<llm-code-output>\n32.0\n</llm-code-output>\nThus Harold will spend \\boxed{32} dollars on wrapping paper."}
+{"problem": "John is an eccentric millionaire.  He decides to fill his swimming pool with bottled water.  A cubic foot of water is 25 liters.  His pool is 10 feet deep and 6 feet by 20 feet.  A liter of water costs $3.  How much does it cost to fill the pool?", "expected_answer": "90000", "predicted_answer": "900", "error_message": "", "is_correct": true, "generation_type": "without_reference_solution", "dataset": "gsm8k", "generation": "Let's solve this problem using Python's sympy library.\n<|python_tag|>\nimport sympy as sp\n\n# dimensional data\npool_depth = 10\npool_length = 20\npool_width = 6\nwater_liters_per_cubic_foot = 25\nwater_price_per_liter = 3\n\n# volume of water\nvolume_of_water = pool_depth * pool_length * pool_width\n\n# cost of water\ntotal_cost = volume_of_water * water_liters_per_cubic_foot * water_price_per_liter\n\n# convert to dollars\ntotal_cost = total_cost / 100\n\n# show final result\ntotal_cost\n<|eom_id|>\n<|start_header_id|>ipython<|end_header_id|>\n900.0\n<|eot_id|><|start_header_id|>assistant<|end_header_id|>\nThus it would cost \\boxed{900} dollars to fill the pool."}
```

Comment:
Let's probably use some example without code blocks here as we still need to finalize our interface with code generation